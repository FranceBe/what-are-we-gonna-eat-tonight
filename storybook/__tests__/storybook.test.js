import initStoryshots, { shallowSnapshot } from '@storybook/addon-storyshots'

initStoryshots({
  configPath: 'storybook',
  test: shallowSnapshot,
})
