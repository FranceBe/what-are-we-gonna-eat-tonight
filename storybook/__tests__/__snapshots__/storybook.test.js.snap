// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots H1 Default 1`] = `
<ForwardRef(H1component__H1Default)>
   Hello I am a Title 
</ForwardRef(H1component__H1Default)>
`;

exports[`Storyshots SelectionButton Default 1`] = `
<ForwardRef(SelectionButtoncomponent__ButtonsContainer)>
  <ForwardRef(SelectionButtoncomponent__SingleButton)
    isActive={false}
    onClick={[Function]}
  >
    option 1
  </ForwardRef(SelectionButtoncomponent__SingleButton)>
  <ForwardRef(SelectionButtoncomponent__SingleButton)
    isActive={false}
    onClick={[Function]}
  >
    option 2
  </ForwardRef(SelectionButtoncomponent__SingleButton)>
</ForwardRef(SelectionButtoncomponent__ButtonsContainer)>
`;

exports[`Storyshots SelectionButton Option 2 is active 1`] = `
<ForwardRef(SelectionButtoncomponent__ButtonsContainer)>
  <ForwardRef(SelectionButtoncomponent__SingleButton)
    isActive={false}
    onClick={[Function]}
  >
    option 1
  </ForwardRef(SelectionButtoncomponent__SingleButton)>
  <ForwardRef(SelectionButtoncomponent__SingleButton)
    isActive={true}
    onClick={[Function]}
  >
    option 2
  </ForwardRef(SelectionButtoncomponent__SingleButton)>
</ForwardRef(SelectionButtoncomponent__ButtonsContainer)>
`;

exports[`Storyshots StyleGuide Colors 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "StyleGuidestory__StorybookDiv-t49j7l-0",
        "isStatic": false,
        "rules": Array [],
      },
      "displayName": "StyleGuidestory__StorybookDiv",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "StyleGuidestory__StorybookDiv-t49j7l-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <ForwardRef(StyleGuidestory__StorybookTitle)>
    Couleurs
  </ForwardRef(StyleGuidestory__StorybookTitle)>
  <ForwardRef(StyleGuidestory__ColorsContainer)>
    <ColorsComponent />
  </ForwardRef(StyleGuidestory__ColorsContainer)>
</StyledComponent>
`;

exports[`Storyshots StyleGuide Polices 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "StyleGuidestory__StorybookDiv-t49j7l-0",
        "isStatic": false,
        "rules": Array [],
      },
      "displayName": "StyleGuidestory__StorybookDiv",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "StyleGuidestory__StorybookDiv-t49j7l-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <ForwardRef(StyleGuidestory__StorybookTitle)>
    Polices
  </ForwardRef(StyleGuidestory__StorybookTitle)>
  <ForwardRef(StyleGuidestory__PlayfairDisplaytRegular)>
     Ceci est la police 1 Regular 
  </ForwardRef(StyleGuidestory__PlayfairDisplaytRegular)>
  <br />
  <ForwardRef(StyleGuidestory__PlayfairDisplayBold)>
     Ceci est la police 1 Bold 
  </ForwardRef(StyleGuidestory__PlayfairDisplayBold)>
  <br />
</StyledComponent>
`;
